zookeeper-api/src/main/java/org/apache/helix/zookeeper/impl/client/ZkClient.java

```diff
@@ -62,7 +62,10 @@
   private static Logger LOG = LoggerFactory.getLogger(ZkClient.class);
 
   public static final int DEFAULT_OPERATION_TIMEOUT = Integer.MAX_VALUE;
-  public static final int DEFAULT_CONNECTION_TIMEOUT = 60 * 1000;
+  // Four times of DEFAULT_SESSION_TIMEOUT would give less 1/125 chance of initial connection
+  // to Zookeeper failure assuming one observer in the 5 observer Zookeeper Quorum is down
+  // with one DNS name configured as connection string with 3.4.13+ Zookeeper client version
+  public static final int DEFAULT_CONNECTION_TIMEOUT = 120 * 1000;
```

Comment:
We already have some customers only onboard the zookeeper-api module and not for Helix. My suggestion is not change the default behavior but let it to be config change.

Reply:
+1. This is a public constant, so we need to avoid changing its value.